data/src/main/java/pl/edu/agh/model/User.java

```diff
@@ -1,5 +1,33 @@
 package pl.edu.agh.model;
 
-public class User {
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "user_")
+public class User extends BaseEntity {
+
+    @Column(length = 60)
+    private String email;
+
+    @Column(length = 30)
+    private String firstName;
+
+    @Column(length = 30)
+    private String lastName;
+
+    @OneToMany(mappedBy = "user")
+    private List<UserInProject> projects;
 
```

Comment:
Ja zostałam wychowana w nienawiści do pustych linii na końcu klasy przed `}`, ale mogę zamknąć oczy, jeśli nikt tego nie czuje

Reply:
Uuuu. Na branchu `IO2019TEAM-31` konsekwentnie ich nie ma

Reply:
Czuję, że moje szanse na wygranie tego wzrosły

Reply:
@jplewa +1 ja nie tyle zostałem wychowany w nienawiści co nie lubię sztucznego rozrastania się kodu, puste linie na początku i na końcu klas i metod wg. mnie są zbędne, po to mamy indentację